drivers/node/ssh/ssh.go

```diff
@@ -86,6 +87,29 @@ func (s *SSH) IsUsingSSH() bool {
 	return len(os.Getenv("TORPEDO_SSH_KEY")) > 0 || len(os.Getenv("TORPEDO_SSH_PASSWORD")) > 0
 }
 
+// WaitForValidUptime Wait for valid uptime.
+func (s *SSH) WaitForValidUptime(n node.Node) (interface{}, error) {
+	uptimeCmd := "sudo uptime -s"
+	t := func() (interface{}, bool, error) {
+		out, err := s.doCmd(n, node.ConnectionOpts{
+			Timeout:         1 * time.Minute,
```

Comment:
can you check if you can use RunCommand function, I think it does the same thing

Reply:
changed logic now.